front/lib/api/assistant/global_agents/configurations/dust/dust.ts

```diff
@@ -347,10 +407,12 @@ export function _getDustGlobalAgent(
     : dummyModelConfiguration;
 
   const hasFilesystemTools = dataSourcesFileSystemMCPServerView !== null;
-
   const hasAgentMemory = agentMemoryMCPServerView !== null;
   const hasToolsets = toolsetsMCPServerView !== null;
 
+  // TODO(growth): We need to put a condition to stop showing onboarding tasks after a certain date?
+  const hasOnboarding = onboardingMCPServerView !== null;
```

Comment:
I don't think we want that. These instructions and this server should only appear in the onboarding conversation, I think it should be a JIT server probably

Reply:
Mmmh in any case I think at some point it should go away and hence be a JIT thing and not something that's always there

Reply:
Yes but that's exactly the todo I wrote on the line just before this condition 🙏🏻

Reply:
Yes but in the meantime it's just going to add that server always, which is not what we want right ?

Reply:
We just did not decided yet what are the conditions to end onboarding, because we did not wanted to map the full flow before starting to experiment 😬 
So I'm experimenting and for now since it's an FF that we won't activate on dust, if you're in you're in you're in 😄

Reply:
That definitely, I thought we never wanted to put in the server instruction when to use the tool and here's it's a bit blurry when and how so I went in there, but can definitely move most of it to the server instruction!

Reply:
(IMHO it's a bit early to have diff in dust.ts)